packages/istanbul-reports/lib/html/index.js

```diff
@@ -132,7 +132,7 @@ class HtmlReport extends ReportBase {
         const linkPath = nodePath.map(ancestor => {
             const target = this.linkMapper.relativePath(node, ancestor);
             const name = ancestor.getRelativeName() || 'All files';
-            return '<a href="' + target + '">' + name + '</a>';
+            return '<a href="' + (target == '' ? '/' + name : target) + '">' + name + '</a>';
```

Comment:
I'm unclear on this change, if target is empty then we would use `name`?  This means we could end up with `<a href="All files">All files</a>`.

Reply:
This is not needed if my previous change is accepted.